centaur/src/main/resources/standardTestCases/dollars_in_strings/dollars_in_strings.wdl

```diff
@@ -0,0 +1,32 @@
+workflow read_dollared_strings {
+
+  call dollars_in_strings
+
+  String dollar = "$"
```

Comment:
Is there any sense testing escaped dollars? Those should definitely not interpolate, right?

Reply:
I believe the whole dance with the `String dollar = "$"` is necessary because there is no escaping dollars in WDL. I might be mis-remembering though